datahub-web-react/src/app/entity/shared/tabs/Dataset/Validations/contract/proposal/DataContractProposal.tsx

```diff
@@ -0,0 +1,274 @@
+import React, { useEffect } from 'react';
```

Comment:
We can remove all proposal files. And in `DataContractTab` we also do not need proposal modal